scala/scala.bzl

```diff
@@ -92,13 +112,16 @@ rm -rf {out}_tmp
       manifest=ctx.outputs.manifest.path,
       jar=ctx.file._jar.path,
       ijar=ctx.file._ijar.path,
-      jars=":".join([j.path for j in jars]),)
+      jars=":".join([j.path for j in jars]),
+      srcjar_cmd = srcjar_cmd
```

Comment:
definitely need a pass of garbage collection, especially around the deps stuff I think I am not fully leveraging the work you did as well. that's gonna be the next major pass at this, I think... getting rid of srcjars and properly leveraging srcs and passing through deps.